doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -58,9 +58,9 @@ Set-up ACRN on your device
 **************************
 
 - Follow the instructions in :Ref:`gsg` to build ACRN using the
-  ``hybrid`` scenario. Here is the build command-line for the `Intel NUC Kit NUC11TNBi5 <https://ark.intel.com/content/www/us/en/ark/products/205596/intel-nuc-11-pro-board-nuc11tnbi5.html>`_::
+  ``hybrid`` scenario. Here is the build command-line for the `Intel Vecow Kit Vecow SPC-7100 <https://marketplace.intel.com/s/offering/a5b3b000000PReMAAW/vecow-spc7100-series-11th-gen-intel-core-i7i5i3-processor-ultracompact-f>`_::
 
-   make clean && make BOARD=nuc11tnbi5 SCENARIO=hybrid
+   make clean && make BOARD=tgl-vecow-spc-7100-Corei7 SCENARIO=hybrid
```

Comment:
The "Update Ubuntu GRUB" section below still mentions the NUC instead of Vecow. Do you plan to update for Vecow?

Reply:
... and the references to ``misc/config_tools/data/nuc11tnbi5/hybrid.xml`` aren't right

Reply:
update